### ISSUE TITLE
"whitespace" rule forces space when using import('lib').SomeType

### ISSUE THREAD

**Issue by AviVahl** [palantir/tslint#4031]
### Bug Report

TypeScript added the ability to dynamically import types using `import()`. Whitespace rule doesn't handle these, and forces a space between the `import` and its parenthesis.

- __TSLint version__: 5.10.0
- __TypeScript version__: 2.9.2
- __Running TSLint via__: (pick one) CLI / VSCode

#### TypeScript code being linted

```ts
let someVar: import('lib').SomeType
```

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint:recommended",
        "tslint-react"
    ],
    "jsRules": {},
    "rules": {
        "ordered-imports": false,
        "quotemark": [
            true,
            "single",
            "jsx-double"
        ],
        "semicolon": [
            true,
            "never"
        ],
        "trailing-comma": false,
        "variable-name": [
            true,
            "allow-leading-underscore",
            "allow-pascal-case"
        ],
        "object-literal-sort-keys": false
    },
    "rulesDirectory": []
}
```

#### Actual behavior
Rule expects space after import keyword. e.g. `import ('lib')`

#### Expected behavior
No space should be forced after `import()`, even when used to bring in types.

**Comment by AlCalzone** [comment]
=> https://github.com/palantir/tslint/pull/3992

**Comment by johnwiseheart** [comment]
Dupe of https://github.com/palantir/tslint/issues/3987
